package.json

```diff
@@ -64,18 +66,17 @@
         "dom-align": "1.12.3",
         "lottie-web": "5.8.1",
         "memoize-one": "6.0.0",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
         "react-flip-toolkit": "7.0.13",
         "react-is": "18.1.0",
         "resize-observer-polyfill": "1.5.1",
         "scroll-into-view-if-needed": "2.2.25",
         "shallowequal": "1.1.0"
     },
     "peerDependencies": {
-        "@types/react": "17.0.39",
-        "@types/react-dom": "17.0.11",
```

Comment:
Seems safe to remove the types from peer deps? though possibly incorrect types would get passed to the octuple components, causing issues? Thoughts?

Reply:
Should be okay as a regular dep, as Octuple is already compiled to js in the host.